### ISSUE TITLE
E2E Encryption des Kontaktformulars

### ISSUE THREAD

**Issue by piepmatz** [Strassengezwitscher/Strassengezwitscher#126]
Heise hat für die Whisteblowing-Page unter https://www.heise.de/tippgeber/kontakt/ was Nettes gebaut, was den Text noch im Browser GPG-verschlüsselt. Die Nutzung scheint ausgesprochen einfach zu sein.

**Comment by Bensk1** [comment]
Kommt man da an den Source Code?

**Comment by piepmatz** [comment]
Ja, das meinte ich mit Nutzung, Nutzung für uns. Das ist eine JS-Lib und ein paar Zeilen Custom JS, die die Lib aufrufen.
Ein Nachteil ist, dass der Public Key dann im Frontend vorhanden sein muss. Aus diesem lässt sich die mutmaßliche Empfängeremailadresse extrahieren, was sich für Spam usw. nutzen ließe.
